### DISCUSSION TITLE
Error message in running tutorial code

### DISCUSSION THREAD

**Discussion by 64Zone** [EpiModel/EpiModel#570] [EpidemicMethods] [answered]
Hello! Hope all is well.

I am trying to run the tutorial code in: EpiModel: An R Package for Mathematical Modeling of Infectious Disease over Networks.

However, I always got error messages when running this line of code (p. 26 of the JSS article): sim2 <- netsim(est2, param, init, control)

Here is the error message: 
_Starting Network Simulation...A ERROR occured in module 'infection.FUN' at step 2: 
	Error in get_attr(dat, "group"): There is no attribute called `group` in the attributes list of the Master list object (dat)
Error in get_attr(dat, "group") : 
  There is no attribute called `group` in the attributes list of the Master list object (dat)_

I have tried to use the EpiModel v1.6.5. but the error stays the same between the old and new versions. Could you share thoughts on how to solve it? Thanks much!

**Comment by martinamorris** [comment, marked as answer]
Please include the output of the r command `sessionInfo()` so that we know what versions of the packages you are using.

**Reply by 64Zone** [reply]
Thanks for the fast response! I just ran the code again and this time it worked. I guess the last time was because I did not restart R and so the old version EpiModel package was not fully loaded?

**Reply by martinamorris** [reply]
Hard to say.  But glad to hear it's working now
